app/code/Magento/Inventory/Plugin/InventoryApi/StockSourceLinksDelete/PreventDelete/DefaultStockPlugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Plugin\InventoryApi\StockSourceLinksDelete\PreventDelete;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\InventoryApi\Api\StockSourceLinksDeleteInterface;
+use Magento\InventoryCatalog\Api\DefaultStockProviderInterface;
+
+/**
+ * Prevent deleting links related to default stock.
+ */
+class DefaultStockPlugin
```

Comment:
Provide more clear name
What does this plugin do?

Reply:
@naydav, renamed.